worlds/subnautica/docs/setup_en.md

```diff
@@ -18,20 +18,26 @@
 
 4. Start Subnautica. You should see a Connect Menu in the topleft of your main Menu.
 
-## Troubleshooting
+## Connecting
 
-If you don't see the connect window check that you see a qmodmanager_log-Subnautica.txt in Subnautica, if not
-QModManager4 is not correctly installed, otherwise open it and look
-for `[Info   :   BepInEx] Loading [Archipelago 1.0.0.0]`, version number doesn't matter. If it doesn't show this, then
-QModManager4 didn't find the Archipelago mod, so check your paths.
+Using the Connect Menu in Subnautica's Main Menu you enter your connection info to connect to an Archipelago Multiworld.
+Menu points:
```

Comment:
I think a small screenshot or two would be ideal.
![image](https://user-images.githubusercontent.com/49286967/182435567-35bcb828-e2f1-4652-8138-9e83d608ab85.png)
![image](https://user-images.githubusercontent.com/49286967/182435589-8b87f277-92e6-4562-a603-eb9980aad035.png)